app/src/main/java/nu/yona/app/ui/settings/SettingsFragment.java

```diff
@@ -183,7 +179,7 @@ public void onError(Object errorMessage) {
                 }
             });
         } catch (Exception e) {
-            AppUtils.throwException(SettingsFragment.class.getSimpleName(), e, Thread.currentThread(), null);
+            AppUtils.reportException(SettingsFragment.class.getSimpleName(), e, Thread.currentThread(), null);
             showAlert(e.toString(), false);
```

Comment:
This is tricky. The exception will likely be a functional one, at least in some cases. Exception handling needs to be selective here, so we only report it to Crashlytics when it is not a user error.

Reply:
removed the reporting part as user is already notified about the error